sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DoNotHardcodeCredentials_DefaultValues.cs

```diff
@@ -8,6 +8,7 @@ namespace Tests.Diagnostics
     class Program
     {
         public const string DBConnectionString = "Server=localhost; Database=Test; User=SA; Password=Secret123";    // Noncompliant
+        public const string EditPasswordPageUrlToken = "{Account.EditPassword.PageUrl}"; // Compliant
```

Comment:
The FN was previously fixed but I've added a test case to avoid regressions.

Reply:
I guess it's FP

Reply:
FP indeed 😄

Reply:
at least the PR title is correct this time 😄

Reply:
and now I see that you've change it